tests/integrationv2/test_session_resumption.py

```diff
@@ -102,3 +100,213 @@ def test_session_resumption_s2n_client(managed_process, cipher, curve, protocol,
         assert results.exception is None
         assert results.exit_code == 0
         assert results.stdout.count(bytes("6 server accepts that finished".encode('utf-8')))
+
+
+@pytest.mark.uncollect_if(func=invalid_test_parameters)
+@pytest.mark.parametrize("cipher", TLS13_CIPHERS, ids=get_parameter_name)
+@pytest.mark.parametrize("curve", ALL_TEST_CURVES, ids=get_parameter_name)
+@pytest.mark.parametrize("certificate", ALL_TEST_CERTS, ids=get_parameter_name)
+@pytest.mark.parametrize("protocol", [Protocols.TLS13], ids=get_parameter_name)
+@pytest.mark.parametrize("provider", [OpenSSL], ids=get_parameter_name)
+def test_tls13_session_resumption_s2n_server(managed_process, tmp_path, cipher, curve, protocol, provider, certificate):
+    port = str(next(available_ports))
+
+    # Use temp directory to store session tickets
+    p = tmp_path / 'ticket.pem'
+    path_to_ticket = str(p)
```

Comment:
We use `path_to_ticket` in multiple locations in this file and since it is a constant, we can define it as a global.

Reply:
See https://github.com/aws/s2n-tls/pull/2814#discussion_r634800735